src/group_theory/order_of_element.lean

```diff
@@ -173,18 +234,42 @@ end
 
 @[simp] lemma order_of_one : order_of (1 : α) = 1 :=
 begin
-apply le_antisymm,
-{ exact order_of_le_of_pow_eq_one (nat.one_pos) (pow_one 1) },
-{ exact nat.succ_le_of_lt ( order_of_pos' ⟨1, ⟨nat.one_pos, pow_one 1⟩⟩) }
+  apply le_antisymm,
+  { exact order_of_le_of_pow_eq_one (nat.one_pos) (pow_one 1) },
+  { exact nat.succ_le_of_lt ( order_of_pos' ⟨1, ⟨nat.one_pos, pow_one 1⟩⟩) }
+end
+
+@[simp] lemma add_order_of_zero : add_order_of (0 : H) = 1 :=
+begin
+  rw ← order_of_of_add_eq_add_order_of,
+  exact order_of_one,
 end
```

Comment:
```suggestion
by rw [←order_of_of_add_eq_add_order_of, of_add_zero, order_of_one]
```
if you're trying to avoid defeq (which is what `exact` is doing). If you are okay with using simp, then it is just a
```
by simp [←order_of_of_add_eq_add_order_of]
```

and similarly below